docs/dev/integrate_operator_how_to.md

```diff
@@ -0,0 +1,53 @@
+## How to add a new operator to the installer?
+
+### Goals:
+To follow this document and be able to
+ - Get an operator’s configuration files be created for installation
+ - Get an operator’s manifest files be put in place for cluster to pick up
+ - Get the config files and manifest files be created as resources in the new cluster, in the appropriate order
+
+### Details
+The source code for the installer is here: https://github.com/openshift/installer. We will refer to this repo as installer-src for this document.
+
+One can can classify all the manifest/config files of an operator in two categories:
+ - Static
+ - Templatized
+ 
+The static ones are clearly just byte blobs and the templates are ones that have some variable that needs to be filled up. Typically the variables that need filling up will come from user preferences as specified in the install config (e.g. cluster name, cluster domain, service cidr etc). Other dependencies could be TLS cert keys for example.
+
+#### The recommended way
+Take the static files and ready it for the Cluster Version Operator (CVO) to pick it up as its payload. See this document:
+https://docs.google.com/document/d/1CGZVEyuloZ9oD4NUArW6dnEpi0WFc6BP2tqPSwFZTCY/edit#heading=h.1zgrwxmpgxbr
+
+Upshot of the above document is that the operator should keep its static manifest files in manifests/ folder of the operator repository and manage image references through an image-references file. The order of creation of the files is alphabetical, so the files should be numbered like below to effectively create the service account before the deployment.
+:
+```
+0000_70_my-operator-name_04_service-account.yaml
+0000_70_my-operator-name_05_deployment.yaml
+```
+where, 0000 is the global run-level, 70 is the operator level, and 04/05 is the internal ordering of the resource manifests.
+
+*What to do for my dynamic template files?*
+
+An operator should auto-discover the install config rather than expand the templates through the installer-src integration (see alternative). Simply use the apiserver access to get the install-config as a config map. The config map is stored by the name ‘cluster-config-v1’ in the ‘kube-system’ namespaces.
```

Comment:
I think it's worth spending a bit more time (wherever these "standard approach" docs live), talking about how this wires in.  As I understand it, you'd take the `/manifests` route with enough static content to get your operator launched, and then you pull everything else you need (e.g. the install config) live from well-known locations.  You focus on the last part of that here, and the Google Doc focuses on the first part.  I don't see anywhere that's tying them together, although the MCO link you have below probably gives a good example for folks who want to read the source ;).

Reply:
I have done a bit of a shift up to tie the two methods (one for static, the other for templates). Review again please.

Reply:
> I have done a bit of a shift up to tie the two methods (one for static, the other for templates). 

This paragraph looks largely unchanged from 4f66f97 -> 2650b11.  I was thinking of an outline more like:

> ## The recommended way
>
> ...Everyone following this integration path will *need* static manifests/configs (as discussed in the Google Doc) to launch their operator...
>
> ### Dynamic configuration
>
> Some operators will need dynamic configuration to accomplish their task.  These operators should set up their static manifests/configs to launch their operator, and then have their operator fill in its configuration by pulling information from cluster resources.  For example, you can pull the install-config as a config map:
>
> ...